fuel-merkle/Cargo.toml

```diff
@@ -24,12 +25,16 @@ datatest-stable = "0.1"
 fuel-merkle-test-helpers = { path = "test-helpers" }
 hex = "0.4"
 rand = "0.8"
+serde_json = "1.0"
 serde_yaml = "0.9"
 
 [features]
 default = ["std"]
 std = ["digest/default", "hex/default", "sha2/default"]
 test-helpers = []
+serde = [
```

Comment:
`fuel-vm` and `fuel-tx` crates has `server` feature too. Could you add `fuel-merkle/serde` there too please?=)
<img width="1325" alt="image" src="https://github.com/FuelLabs/fuel-vm/assets/18346821/519cbda5-d468-47a4-a516-90a59d2d63e0">